### PR TITLE
Show the documentation structure

### PR DESCRIPTION
**Pull request by akaszynski**
Show the documentation structure in `structure.rst`.

### CODE REVIEW

**Comment by akaszynski** [comment]
@jorgepiloto, if you agree, please add `AUTHORS.md` to https://github.com/ansys/ansys-templates/

**Comment by phxnsharp** [comment]
I'm still philosophically working though this one in my head. At Ansys we create products and libraries that are much bigger than individual contributors. Projects should always be done from the perspective that new people will be taking them over, and probably relatively quickly. This encourages sustainable documentation. An author list can be a crutch for not writing proper documentation.

For example, ansys-common-variableinterop was written by a 5 person team and might during any sprint be worked on by another team of similar size. An author list would be long from the start, and unwieldy quickly.  Besides, GitHub already keeps track of who contributes for us.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ repos:
     additional_dependencies: [black==22.12.0]
 
 - repo: https://github.com/pycqa/isort
-  rev: 5.11.4
+  rev: 5.11.5
```
Comment (jorgepiloto) [inline .pre-commit-config.yaml:17]:
```suggestion
  rev: 5.12.0
```

File: doc/source/packaging/structure.rst [hunk 2]
```diff
@@ -201,6 +201,18 @@ This is the preferred way of naming directories and files inside the
...
 
+
+The ``AUTHORS.md`` file
+-----------------------
+An ``AUTHORS.md`` file is used to list the authors of the repository. It is
+used to credit the authors for their individual contributions and provide a
+record of authorship for the codebase. Use your first and last name and provide
+a link to your GitHub username.
+
+.. literalinclude:: code/changelog_file.md
```
Comment (jorgepiloto) [inline doc/source/packaging/structure.rst:212]:
```suggestion
.. literalinclude:: code/authors_file.md
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]

**Comment by jorgepiloto** [comment]
Thanks for enhancing this, @akaszynski.

Your answer got me thinking, @phxnsharp and I decided to open [this disucussion](https://github.com/pyansys/dev-guide/discussions/250) so we can agree about the usage and content of the `AUTHORS` file. The discussion presents some exmaples from the GNU project and Google.

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**